sirepo/package_data/static/html/heatplot.html

```diff
@@ -18,6 +18,8 @@
           </g>
           <path class="line line-amr-grid"></path>
         </g>
+      <g class="radia-watermark-group">
```

Comment:
`radia` should be `sr`. This is a Sirepo watermark. `radia` also conflicts with the name of the code.

I do not think `-group` is necessary.

Reply:
1360b65

It's a RadiaSoft watermark according to the example in the issue.  I changed to "rs-" to get away from "radia", but I want to keep "-group" to distinguish it from "-icon"